app/src/main/java/com/example/fragmentassg2/fragments/FragmentLogin.kt

```diff
@@ -56,14 +87,20 @@ class FragmentLogin : Fragment(), View.OnClickListener {
         }
     }
     private fun login(){
-        val number ="9953755955"
 
-        if(editPhone.editText?.text.toString()==number){
+        val checkPhoneNumber:Boolean=viewModel.validatePhoneNumber(Constant.USER_PHONE_NUMBER)
+        val phoneNumberEqual:Boolean=viewModel.phoneEqual(Constant.USER_PHONE_NUMBER)
+        val phoneNumberLength:Boolean=viewModel.phoneLength(Constant.USER_PHONE_NUMBER)
+
+
+
+        if(phoneNumberEqual && checkPhoneNumber && phoneNumberLength){
```

Comment:
@raghav32 make a single function in viewmodel and return a boolean value

Reply:
Also, currently your logic is not working properly, Right now it is moving me to next screen on any 10 digit number